content/_partials/downloadlist.html.haml

```diff
@@ -1,42 +1,105 @@
-    %span.icon 
-    %span.title
-      Windows
-  .dropdown-divider
-  %a.dropdown-item{:href=>'http://mirrors.jenkins-ci.org/windows/latest'} Plugins
\ No newline at end of file
+  .row
+    .col-md-6.text-xs-center
+      %h3 LTS Release
+      %p.details
+        %a.item{:href=>'http://jenkins-ci.org/changelog'} Changelog
+        |
+        %a.item{:href=>'http://mirrors.jenkins-ci.org/war/'} Past Releases
+
+      %p LTS or Long-term support releases are chosen by the community every 12 weeks from the steady stream of regular releases as the major/stable release for that period.
+      
+      %p
+        .btn-group
+          %a.btn.btn-primary{:href=>'http://mirrors.jenkins-ci.org/war/latest/jenkins.war'}
+            %i.icon-download  1.625.3 .war
+          %button.btn.btn-primary.dropdown-toggle{'data-toggle' => 'dropdown', 'aria-haspopup'=>"true", 'aria-expanded'=>"false"}
+          .dropdown-menu
+            %a.dropdown-item{:href=>'https://registry.hub.docker.com/u/jenkinsci/jenkins/'}
+              %span.icon 
+              %span.title
+                Docker
+            %a.dropdown-item{:href=>'http://pkg.jenkins-ci.org/debian/'}
+              %span.icon 
+              %span.title
+                FreeBSD
+            %a.dropdown-item{:href=>'http://pkg.jenkins-ci.org/debian/'}
+              %span.icon 
+              %span.title
+                Gentoo
+            %a.dropdown-item{:href=>'http://pkg.jenkins-ci.org/debian/'}
+              %span.icon 
+              %span.title
+                Mac OSX
+            %a.dropdown-item{:href=>'http://pkg.jenkins-ci.org/debian/'}
+              %span.icon 
+              %span.title
+                OpenBSD
+            %a.dropdown-item{:href=>'http://pkg.jenkins-ci.org/debian/'}
+              %span.icon 
+              %span.title
+                OpenSUSE
+            %a.dropdown-item{:href=>'http://pkg.jenkins-ci.org/debian/'}
+              %span.icon 
+              %span.title
+                Red Hat/Fedora/CentOS                 
+            %a.dropdown-item{:href=>'http://pkg.jenkins-ci.org/debian/'}
+              %span.icon 
+              %span.title
+                Ubuntu/Debian
+            %a.dropdown-item{:href=>'http://mirrors.jenkins-ci.org/windows/latest'} 
+              %span.icon 
+              %span.title
+                Windows    
+      
+    .col-md-6.text-xs-center
+      %h3 Weekly Release
+      %p.details
+        %a.item{:href=>'http://jenkins-ci.org/changelog'} Changelog
+        |
+        %a.item{:href=>'http://mirrors.jenkins-ci.org/war/'} Past Releases
+
+      %p The Jenkins community produces a new release weekly to deliver bug fixes and new features rapidly to users and plugin developers who need them.
+      
+      %p
+        .btn-group
+          %a.btn.btn-primary{:href=>'http://mirrors.jenkins-ci.org/war/latest/jenkins.war'}
+            %i.icon-download  1.644 .war
```

Comment:
FWIW there's a variable with the latest version which should be inserted here, `site.jenkins.latest`